sources/Context.h

```diff
@@ -106,6 +106,15 @@ class Context {
    */
   void exportOutputs();
 
+  /**
+   * @brief Export results file
+   *
+   * @param isOk the simulation was ok
```

Comment:
Maybe we could call this parameter "simulationOk"?

Reply:
we can but the name of the parameter in the json needs to be isOk